README.md

```diff
@@ -212,6 +230,63 @@ $ curl localhost:9200/_cat/indices
 green open ks-logstash-log-2021.04.06 QeI-k_LoQZ2h1z23F3XiHg  5 1 404879 0 298.4mb 149.2mb
 ```
 
+### Fluentd 
+
+Fluentd acts as a forward log layer to collect logs from fluentbit and other Apps. 
+
+#### Collect logs from Fluentbit
+
+##### Enable Fluentbit forward plugin
+
+At first, we should enable the forward plugin in fluentbit to send logs to fluentd.
+
+```shell
+kubectl apply -f manifests/fluentd/fluentbit-output-forward.yaml
+```
+
+##### Fluentd ClusterFluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-cluster-cfg-output-es.yaml
+```
+
+##### Fluentd FluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-namespaced-cfg-output-es.yaml
+```
+
+##### Fluentd mixed FluentdConfig and ClusterFluentdConfig example
+
+```shell
+kubectl apply -f manifests/fluentd/fluentd-mixed-cfgs-output-es.yaml
+```
+
+##### Fluentd ClusterOutput and Output example in multi-tenant scenario
```

Comment:
Combining Fluentd cluster-wide output and namespace-wide output for the multi-tenant scenario